Dockerfile

```diff
@@ -26,9 +26,9 @@ ENV BSC_HOME=/bsc
 ENV HOME=${BSC_HOME}
 ENV DATA_DIR=/data
 
-ENV PACKAGES ca-certificates~=20220614 jq~=1.6 \
-  bash~=5.1.16-r2 bind-tools~=9.16.29-r0 tini~=0.19.0 \
-  grep~=3.7 curl~=7.83.1-r2 sed~=4.8-r0  curl~=7.83
+ENV PACKAGES ca-certificates~=20220614-r0 jq~=1.6 \
+  bash~=5.1.16-r2 bind-tools~=9.16.33-r0 tini~=0.19.0 \
+  grep~=3.7 curl==7.83.1-r3 sed~=4.8-r0
```

Comment:
for the `curl`, please do not point a must version, please use `~=`

Reply:
fixed